chokidar/index.d.ts

```diff
@@ -5,20 +5,24 @@
 
 /// <reference types="node" />
 
-declare module "fs"
+declare module "chokidar"
 {
-    interface FSWatcher
+    export class FSWatcher
     {
+        constructor(options?: WatchOptions);
         add(fileDirOrGlob:string):void;
         add(filesDirsOrGlobs:Array<string>):void;
         unwatch(fileDirOrGlob:string):void;
         unwatch(filesDirsOrGlobs:Array<string>):void;
         getWatched():any;
+        on(event: 'add', fn: (path: string, stats?: fs.Stats) => void): this;
+        on(event: 'change', fn: (path: string, stats?: fs.Stats) => void): this;
+        on(event: 'unlink', fn: (path: string) => void): this;
+        on(event: 'raw', fn: (event: Event, path:string, details:any) => void): this;
```

Comment:
@andy-ms Looking at https://github.com/paulmillr/chokidar/blob/master/lib/nodefs-handler.js#L26 it seems the raw event from node fs is used. According to https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/node/node-4.d.ts#L1661 that seems to be a regular `event: string`.

Since this issue is blocking us from progressing on with typed chokidar, could you remove the `Event` from this file? Thank you in advance!